tests/docker/Dockerfile

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 
 # The base image of openjdk:17 is typically oraclelinux:8-slim, which doesn't include apt-get. 
-# Therefore, use openjdk:17-buster instead.
-ARG jdk_version=openjdk:17-buster
+# Therefore, use openjdk:17-bullseye instead.
+ARG jdk_version=openjdk:17-bullseye
```

Comment:
Should we remove the default value here since we are taking that from the build script where we also have a default value?

If we also update the README.md using a placeholder (e.g. <JDK_IMAGE>), in the next base image update we would only need to update the default image name in one place. Wdyt?

Reply:
sounds good